### PR TITLE
feat: allow multiple webhook body sources

### PR DESCRIPTION
**Pull request by zepatrik**
## Related issue(s)

Currently the webhook body can only be loaded from a local file. This allows to use any source enabled in the JSONSchema loaders (file, base64, https, ...)

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: selfservice/hook/web_hook_test.go [hunk 1]
```diff
@@ -91,47 +91,78 @@ func TestApiKeyInCookieStrategy(t *testing.T) {
...
+					"Some-Header": []string{"Some-Value"},
+				},
+				RequestMethod: "POST",
+				RequestUrl:    "https://test.kratos.ory.sh/some-test-path",
+				Identity:      i,
+			},
+		},
+		{
+			desc:     "filepath without scheme",
+			template: "./stub/test_body.jsonnet",
```
Comment (aeneasr) [inline selfservice/hook/web_hook_test.go:121]: Would it make sense to emit a warning if this is used, and at some point deprecate it (and create an issue tracking the deprecation)?

File: driver/config/.schema/config.schema.json [hunk 2]
```diff
@@ -147,7 +147,13 @@
             },
             "body": {
               "type": "string",
-              "description": "Path to the jsonnet template used for payload generation. Only used for those HTTP methods, which support HTTP body payloads"
+              "description": "URI pointing to the jsonnet template used for payload generation. Only used for those HTTP methods, which support HTTP body payloads",
+              "examples": [
+                "file:///path/to/body.jsonnet",
+                "/path/to/body.jsonnet",
+                "./body.jsonnet",
+                "base64://ZnVuY3Rpb24oY3R4KSB7CiAgaWRlbnRpdHlfaWQ6IGlmIGN0eFsiaWRlbnRpdHkiXSAhPSBudWxsIHRoZW4gY3R4LmlkZW50aXR5LmlkLAp9="
```
Comment (aeneasr) [inline driver/config/.schema/config.schema.json:155]: Do we allow loading from `http` too?

File: selfservice/hook/web_hook_test.go (hunk 1 above)
Comment (zepatrik) [reply selfservice/hook/web_hook_test.go:121]: I mean we allow it for all other URIs like identity schema as well in the same way...

File: driver/config/.schema/config.schema.json (hunk 2 above)
Comment (zepatrik) [reply driver/config/.schema/config.schema.json:155]: Yes, if the loader is registered.

File: selfservice/hook/web_hook_test.go (hunk 1 above)
Comment (aeneasr) [reply selfservice/hook/web_hook_test.go:121]: I am actually not sure if we support filepaths in the other things (e.g. jsonnet loader). If we don't I think we should deprecate, if we do, we can keep it

Comment (zepatrik) [reply selfservice/hook/web_hook_test.go:121]: Ah right, missed that we have a different loader there. Will mark it as deprecated then.

**Review by aeneasr — APPROVED** [review]

**Pull request merged**